### ISSUE TITLE
Cups options, quote escaping

### ISSUE THREAD

**Issue by khobbits** [leoarnold/puppet-cups#32]
**Given** Centos 7.4 with Puppet 4.10.9 and CUPS  1.6.3.

**When** I apply the manifest

```puppet
        class { '::cups':
            default_queue => 'WMS_Global_Printing',
        }

        cups_queue { "WMS_Global_Printing":
            ensure      => "printer",
            uri         => "smb://mill-print/WMS_Global_Printing",
            description => "The Mill London Global Print Queue",
            model       => "drv:///sample.drv/generic.ppd",
            accepting => true,
            enabled   => true,
            options => {
                "auth-info-required"   => "username,password"
            }
        }  
```

**Then** I get this notice message every run:

```Shell
Notice: /Stage[main]/Mill3d_base::Papercut/Cups_queue[WMS_Global_Printing]/options: options changed '{"auth-info-required"=>"\"username,password\""}' to '{"auth-info-required"=>"username,password"}'
```

**Expectation** I only see a notice if the settings have changed, rather than each puppet run.

**Comment by leoarnold** [comment]
This issue should be fixed in v2.0.3 which was just published.

**Comment by khobbits** [comment]
Confirmed, seems to fix the problem.  Thanks.
